src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -97,12 +223,13 @@ def _check_fixed_for_custom_binary(testcase, job_type, testcase_file_path):
         command, symbolized_crash_stacktrace, unsymbolized_crash_stacktrace)
     testcase.last_tested_crash_stacktrace = data_handler.filter_stacktrace(
         stacktrace)
-    data_handler.update_progression_completion_metadata(
-        testcase,
-        revision,
-        is_crash=True,
-        message='still crashes on latest custom build')
-    return
+    progression_task_output = uworker_io.ProgressionTaskOutput(
+        custom_binary=True, crash_on_latest=True, crash_revision=revision)
```

Comment:
I was not sure if `build_manager.is_custom_binary()` needed `setup.setup_testcase` to be called before so that the env has all the needed variables. 
Otherwise, we could probably look for the related environment variables in the `uworker_env` directly in preprocess. WDYT?